knowledge-base/actions/megalinter/megalinter/action-security.yml

```diff
@@ -0,0 +1,7 @@
+name: "MegaLinter, by OX Security"
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
+    pull-requests: read
+    pull-requests-reason: to skip duplicate actions
+    #Reference: https://github.com/oxsecurity/megalinter/blob/09bd8a3aab852fd55f80b3c66dab3cc360c0b5b0/mega-linter-runner/lib/upgrade.js
```

Comment:
@RksRocks update reference to https://github.com/oxsecurity/megalinter/blob/f85e07ebdf81d8db3a5fbaf4eadfd00aca7a3a64/megalinter/reporters/GithubCommentReporter.py#L85